serialized_data_converter.py

```diff
@@ -349,6 +349,7 @@ def convert(self, edit):
                 self.output = yaml.yaml_dumps(
                     self.plist,
                     default_flow_style=default_flow_style,
+                    indent=self.settings.get("yaml_indent"),
```

Comment:
There is a default in source, at https://github.com/facelessuser/SerializedDataConverter/pull/9/files#diff-31432931070928712986b2b26f7403e8R233